content/shmem_ctx.tex

```diff
@@ -64,6 +64,13 @@ \subsubsection{\textbf{SHMEM\_CTX\_CREATE}}
 
         \apitablerow{\CONST{SHMEM\_CTX\_PRIVATE}}{
             The given context will be used only by the thread that created it.}
+
+        \apitablerow{\CONST{SHMEM\_CTX\_NOSTORE}}{
+            Quiet and fence operations performed on the given context are not
```

Comment:
The previous two options are requirements on the applications, and the last one is on the library. To make it clear, consider change to the sentence : " When Quiet and fence operations are performed on the given context, the implementation is not required to enforce ... "